accname/name/comp_tooltip.html

```diff
@@ -11,11 +11,38 @@
 </head>
 <body>
 
-  <p>Tests the <a href="https://w3c.github.io/accname/#comp_tooltip">#comp_tooltip</a> portions of the AccName <em>Name Computation</em> algorithm.</p>
+<p>Tests the <a href="https://w3c.github.io/accname/#comp_tooltip">#comp_tooltip</a> portions of the AccName <em>Name Computation</em> algorithm.</p>
 
-<a href="#" title="label" data-expectedlabel="label" data-testname="link label from tooltip" class="ex"><img src="#" alt=""></a>
+<a href="#" title="label" data-expectedlabel="label" data-testname="link with img with tooltip label" class="ex"><img src="#" alt=""></a>
+<a href="#" title="label" data-expectedlabel="label" data-testname="link with text with tooltip label" class="ex">content</a>
+<div title="label" data-expectedlabel="label" data-testname="div with text with tooltip label" class="ex">content</div>
+<img src="foo.jpg" title="label" data-expectedlabel="label" data-testname="img with tooltip label without alt" class="ex">
+<img src="foo.jpg" title="label" data-expectedlabel="broken" alt="broken" data-testname="img with tooltip label with alt" class="ex">
+<img src="foo.jpg" data-expectedlabel="broken" alt="broken" data-testname="img with tooltip label without title" class="ex">
```

Comment:
Would be good to test explicitly empty alt too (`alt="" label="title"`)

Reply:
Done in https://github.com/web-platform-tests/wpt/pull/42093/commits/0f8aef29e3f3ef6a79706535ca5a53e30f1e8880